benches/crit_bench.rs

```diff
@@ -297,32 +324,55 @@ fn bench_frame_compression_throughput(c: &mut Criterion) {
                 })
             },
         );
+        // group.bench_with_input(
```

Comment:
there are a lot of commented benchmarks

Reply:
Added by mistake to the pr. I'll revert these once its ready to merge.